src/pinning.js

```diff
@@ -148,6 +162,32 @@ class Pinning {
     })
   }
 
+  async getConfig (address) {
+    return new Promise((resolve, reject) => {
+      const spacesFromRoot = address => {
+        const config = this.openDBs[address].db
+          .iterator({ limit: -1 })
+          .collect()
+          .reduce((conf, entry) => {
+            const data = entry.payload.value
+            if (data.type === rootEntryTypes.SPACE) {
+              if (!conf.space) conf.spaces = {}
+              const name = data.odbAddress.split('.')[2]
+              conf.spaces[name] = {
+                DID: data.DID
+              }
+            }
+            return conf
+          }, {})
+        resolve(config)
+      }
+      // we need to open substores on replicated, otherwise it will break
+      // the auto pinning if the user adds another store to their root store
+      this.openDB(address, spacesFromRoot, this._openSubStores.bind(this))
+      this.analytics.trackListSpaces(address)
```

Comment:
swap out analytics event for another event for this call or just remove 

even with comment, can't remember why the sub stores need to be opened?

Reply:
nice catch!

Reply:
The openSubStores fn will be called on the replicated event. If it's not present and a user opens a new space the rootStore will replicate but the space will not be opened.